### PR TITLE
docs: Add README

### PR DESCRIPTION
**Pull request by tchx84**

### CODE REVIEW

**Comment by tchx84** [comment]
@rffontenelle I put the _basics_ here, maybe you can help me extend this to cover adding a new language?

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+```
+
+## Steps
+
+```bash
+$ cd Gameeky
+$ python3 docs/tools/update_pot.py docs/basics/src/en/index.md docs/basics/po/gameeky.pot
+$ python3 docs/tools/update_po.py docs/basics/po --language es
+$ python3 docs/tools/update_md.py docs/basics/src/ docs/basics/po/ --language es
+$ python3 docs/tools/update_html.py docs/basics/src/ docs/basics/data/headers.xml docs/basics/po docs/basics/html/ --language es
```
Comment (rffontenelle) [inline docs/README.md:17]: Nit pick:

```suggestion
$ python3 docs/tools/update_html.py docs/basics/src/ docs/basics/data/headers.xml docs/basics/po/ docs/basics/html/ --language es
```

**Review by rffontenelle — COMMENTED** [review]
README.md on root directory also have instructions for UI and docs. How to conciliate that?

**Comment by tchx84** [comment]
> README.md on root directory also have instructions for UI and docs. How to conciliate that?

The right move would be to:

1. Move `./po/` specific intructions to a `./po/README.md`
2. Expand `./docs/README.md/` with mode details.
3. Update the CONTRIBUTING.md file to point to these other two READMEs.

**Pull request merged**